### ISSUE TITLE
(Python) Streaming example issues 

### ISSUE THREAD

**Issue by hhbyyh** [intel-analytics/analytics-zoo#1353]
Most issues in https://github.com/intel-analytics/analytics-zoo/issues/1342 are also in Python.

1. If we don't have a pre-trained model for text classification, I think we should just remove the example.

2. The readme of object detection example should be updated. 
```
In this example, we demo the model inference using a pre-trained SSD model and Spark Streaming. The example contains two programs:
1. ImagePathWriter simulates a streaming source, and periodically write image paths into text files in the image path folder.
2. StreamingObjectDetection continuously monitors the image path folder,  and periodically loads corresponding images, run object detection model on them,  and save the results in the output folder.
"
```

3. image_path_writer command is incorrect. Should use spark submit and the program parameters are wrong. And we should use longer delay and smaller batch by default.

MASTER=...
imageSourcePath=... // image path. Local file system/HDFS/Amazon S3 are supported
streamingPath=... // text files. Local file system/HDFS/Amazon S3 are supported
**${SPARK_HOME}/bin/spark-submit-with-zoo.sh** \
    --master ${MASTER} \
    --driver-memory 5g \
    image_path_writer.py \
    **--streamingPath ${streamingPath} --imageSourcePath ${imageSourcePath}**

**Comment by qiyuangong** [comment]
Fxied. Thanks to @qiuxin2012 .

**Comment by hhbyyh** [comment]
Need to fix parameter imageSourcePath in image_path_writer. Right now it's inconsistent between readme and code.

**Comment by qiyuangong** [comment]
@hhbyyh addressed in #1408 . :)
